pkg/apis/internal.acorn.io/v1/appspec.go

```diff
@@ -51,9 +51,10 @@ func (in Build) BaseBuild() Build {
 type Protocol string
 
 var (
-	ProtocolTCP  = Protocol("tcp")
-	ProtocolUDP  = Protocol("udp")
-	ProtocolHTTP = Protocol("http")
+	ProtocolTCP   = Protocol("tcp")
+	ProtocolUDP   = Protocol("udp")
+	ProtocolHTTP  = Protocol("http")
+	ProtocolHTTPS = Protocol("https")
```

Comment:
Can you create a new type PublishProtocol that has all for of these values. I don’t want https in the protocol list because it’s not supported as a value anywhere else Protocol is used.

Reply:
I actually had this and reverted, because I found it ugly to have those duplicates there :D

Reply:
Brought it back :heavy_check_mark: